src/applications/mhv/medical-records/components/LabsAndTests/EkgDetails.jsx

```diff
@@ -9,7 +11,68 @@ const EkgDetails = props => {
 
   const formattedDate = formatDateLong(record?.date);
 
-  const download = () => {};
+  const generateEkgDetails = async () => {
+    const pdfData = {
+      headerLeft: 'Roberts, Jesse',
+      headerRight: 'Date of birth: January 1, 1970',
+      footerLeft: `Report generated by My HealtheVet and VA on ${moment().format(
```

Comment:
I think this is a different date format than other PDFs (`LL` vs `LLL`) - is that intentional?

Reply:
addressed